gradle.properties

```diff
@@ -17,4 +17,6 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
-#org.gradle.java.home=<path to jdk1.8 for tests>
\ No newline at end of file
+#org.gradle.java.home=<path to jdk1.8 for tests>
+android.useAndroidX=true
```

Comment:
I think it's not relate to current issue

Reply:
Hi, @senneco

Reply:
But this not related to moxy, right?) Could it be placed to target moxy-andoridx-sample module?

Reply:
Sorry, my mistake
I didn't think about another gradle.properties file in for the one module.